portfolio.html

```diff
@@ -27,23 +28,58 @@ <h2>About me</h2>
 		<p class="paragraph">I am a recent graduate of IO academy, eager and motivated to throw myself into work. My degree and this course have given me technical skills in online web creation, video production and coding, whilst allowing me to lead and manage multiple group projects at once. I am reliable, confident and always keen to contribute as part of a team. I am dedicated and committed to any task and never afraid to step forward and lead.</p>
```

Comment:
more semantic name needed i think "paragraph" could refer to text anywhere on the site